pinot-connectors/pinot-spark-connector/src/test/resources/schema/pinot-schema.json

```diff
@@ -0,0 +1,57 @@
+{
```

Comment:
Perhaps we should have a PinotSchema <-> SparkSchema converter?

Reply:
yes